datasets/swda/README.md

```diff
@@ -0,0 +1,254 @@
+- **Point of Contact: [Christopher Potts](https://web.stanford.edu/~cgpotts/)**
+
+### Dataset Summary
+
+The Switchboard Dialog Act Corpus (SwDA) extends the Switchboard-1 Telephone Speech Corpus, Release 2 with 
+turn/utterance-level dialog-act tags. The tags summarize syntactic, semantic, and pragmatic information about the 
+associated turn. The SwDA project was undertaken at UC Boulder in the late 1990s.
+The SwDA is not inherently linked to the Penn Treebank 3 parses of Switchboard, and it is far from straightforward to 
+align the two resources. In addition, the SwDA is not distributed with the Switchboard's tables of metadata about the 
+conversations and their participants.
+
+
+### Supported Tasks and Leaderboards
+
+| Model           | Accuracy  |  Paper / Source | Code |
+| ------------- | :-----:| --- | --- |
+| SGNN (Ravi et al., 2018) | 83.1 | [Self-Governing Neural Networks for On-Device Short Text Classification](https://www.aclweb.org/anthology/D18-1105.pdf)
+| CASA (Raheja et al., 2019) | 82.9 | [Dialogue Act Classification with Context-Aware Self-Attention](https://www.aclweb.org/anthology/N19-1373.pdf)
+| DAH-CRF (Li et al., 2019) | 82.3 | [A Dual-Attention Hierarchical Recurrent Neural Network for Dialogue Act Classification](https://www.aclweb.org/anthology/K19-1036.pdf)
+| ALDMN (Wan et al., 2018) | 81.5 | [Improved Dynamic Memory Network for Dialogue Act Classification with Adversarial Training](https://arxiv.org/pdf/1811.05021.pdf)
+| CRF-ASN (Chen et al., 2018) | 81.3 | [Dialogue Act Recognition via CRF-Attentive Structured Network](https://arxiv.org/abs/1711.05568) | |
+| Bi-LSTM-CRF (Kumar et al., 2017) | 79.2 | [Dialogue Act Sequence Labeling using Hierarchical encoder with CRF](https://arxiv.org/abs/1709.04250) | [Link](https://github.com/YanWenqiang/HBLSTM-CRF) |
+| RNN with 3 utterances in context (Bothe et al., 2018) | 77.34 | [A Context-based Approach for Dialogue Act Recognition using Simple Recurrent Neural Networks](https://arxiv.org/abs/1805.06280) | |
+
+### Languages
+
+The language supported is English.
+
+## Dataset Structure
+
+Utterance are tagged with the [SWBD-DAMSL](https://web.stanford.edu/~jurafsky/ws97/manual.august1.html) DA.
+
+### Data Instances
+
+A|What is the nature of your company's business?|qw
+
+B|Well, it's actually, uh,|^h
+
+B|we do oil well services.|sd
+
+### Data Fields
+
+Utterances are written one per line in the format Speaker | Utterance Text | Dialogue Act Tag.
+Setting the utterance_only_flag == True, will change the default output to only one utterance per line i.e. no speaker or DA tags.
+Utterances marked as Non-verbal ('x' tags) are removed i.e. 'Laughter' or 'Throat_clearing'.
+Utterances marked as Interrupted ('+' tags) and continued later are concatenated to make un-interrupted sentences.
+All disfluency annotations are removed i.e. '#', '<', '>', etc.
```

Comment:
Here we expect a brief description of each data fields (speaker, utterance_text, dialogue_act_tag, dialogue_act_label).

Reply:
fixed it